src/amberscript/parser_shader_test.cc

```diff
@@ -122,6 +122,34 @@ TEST_F(AmberScriptParserTest, ShaderPassThroughExtraParameters) {
   EXPECT_EQ("1: extra parameters after SHADER PASSTHROUGH: INVALID", r.Error());
 }
 
+TEST_F(AmberScriptParserTest, ShaderPassThroughTargetEnv) {
```

Comment:
I guess we will have to change this into negative test, or remove it.

Reply:
done